packages/rum/src/domain/record/trackers/trackMediaInteraction.ts

```diff
@@ -20,16 +21,19 @@ export function trackMediaInteraction(
     [DOM_EVENT.PLAY, DOM_EVENT.PAUSE],
     (event) => {
       const target = getEventTarget(event)
+      if (!target) {
+        return
+      }
+      const id = scope.getSerializedNodeId(target)
       if (
-        !target ||
-        getNodePrivacyLevel(target, configuration.defaultPrivacyLevel) === NodePrivacyLevel.HIDDEN ||
-        !hasSerializedNode(target)
+        id === undefined ||
+        getNodePrivacyLevel(target, configuration.defaultPrivacyLevel) === NodePrivacyLevel.HIDDEN
       ) {
         return
       }
       mediaInteractionCb(
         assembleIncrementalSnapshot<MediaInteractionData>(IncrementalSource.MediaInteraction, {
-          id: getSerializedNodeId(target),
+          id,
```

Comment:
Can `assembleIncrementalSnapshot` accept only an `id` of type `NodeID`, or does our link with rum events format prevents this?